common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_apmanager.yaml

```diff
@@ -201,3 +201,12 @@ cACTION_APMANAGER_WIFI_CREDENTIALS_UPDATE_REQUEST:
 cACTION_APMANAGER_HEARTBEAT_NOTIFICATION:
   _type: class
 
+cACTION_APMANAGER_SUPPORTED_CHANNELS_REQUEST:
+  _type: class
+
+cACTION_APMANAGER_SUPPORTED_CHANNELS_RESPONSE:
+  _type: class
+  supported_channels_list:
+    _type: beerocks::message::sWifiChannel
+    _length: [ "beerocks::message::SUPPORTED_CHANNELS_LENGTH" ]
```

Comment:
I don't like very much that this is a fixed-size list, but it's already done like that in other places so OK.